packages/build/tests/plugins/snapshots/tests.js.md

```diff
@@ -2957,6 +3049,122 @@ Generated by [AVA](https://ava.li).
           origin: config␊
           package: ./plugin`
 
+## No --node-path version validation is performed against the plugin engines if the plugins_system_node_version feature flag is set
```

Comment:
With the FF set by default we're just re-runing the test we previously had a validating that no error logs are outputed.